packages/pixeloven/cli/src/templates/project/package.json.ejs

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "<%= props.projectName %>",
+    "author": "<%= props.projectAuthor %>",
+    "description": "<%= props.projectDescription %>",
+    "private": true,
+    "workspaces": [
+      "apps/*",
+      "packages/*"
+    ],
+    "devDependencies": {
+      "@pixeloven/cli": "6.0.0-rc.0",
+      "@pixeloven/cli-addon-generators": "6.0.0-rc.0",
+      "@pixeloven/cli-addon-storybook": "6.0.0-rc.0",
+      "@pixeloven/cli-addon-webpack": "6.0.0-rc.0",
+      "husky": "4.2.1",
+      "lerna": "3.20.2"
+    },
+    "scripts": {
+      "all:bootstrap": "yarn install && yarn all:compile && yarn all:linker",
```

Comment:
I think I asked this elsewhere, but since we only have one set of these, do we need the `all` prefix?

Reply:
I plan on creating the more short cuts soon.